haystack/components/evaluators/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .answer_exact_match import AnswerExactMatchEvaluator
```

Comment:
Now that the metrics are (in)directly associated with specific upstream components, I don't think we need an additional `statistical` submodule after all.